portfolio/src/main/webapp/script.js

```diff
@@ -12,56 +12,77 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+
+
+const loc = "/images/life/";
+let currImageNum = 1;
+let sliderSpeed=5000;
+
 // Function that shows or hides the list of courses
 // for a particular semester upon user click
 function toggleCourseList(listObject) {
     let semester = listObject.id;
     let semCourseList = "courselist" + semester;
     let lst = document.getElementById(semCourseList);
-    if(lst.style.display === "none" || lst.style.display === "")
+    if(listObject.className === "up")
     {
         lst.style.display = "block";
```

Comment:
Optional: Instead of directly changing the style of the lst element here, you can also do something similar to the line below - change the class name and let the css file take care of styling.